modules/monitoring-creating-new-alerting-rules.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * monitoring/managing-alerts.adoc
+
+:_content-type: PROCEDURE
+[id="creating-new-alerting-rules_{context}"]
+= Creating new alerting rules
+
+As a cluster administrator, you can create new alerting rules based on platform metrics. 
+These alerting rules fire alerts based on the values of chosen metrics.
+
+[NOTE]
+====
+If you create a new, customized `AlertingRule` resource based on an existing platform alerting rule, you should silence the original alert to avoid receiving conflicting alerts.
```

Comment:
```suggestion
If you create a customized `AlertingRule` resource based on an existing platform alerting rule, silence the original alert to avoid receiving conflicting alerts.
```
IBM SG: 
```
Whenever possible, rewrite [should] in a more direct way. Do not use "should" to refer to an action that must be performed. For example, when parameters must be reset, write "Reset the parameters" and not "You should reset the parameters".
```
Our localization team also wrote an excellent blog post about their difficulties interpreting "should."

Reply:
Good catch. Fixed.